src/routes/api/assets/patchById.js

```diff
@@ -1,16 +1,85 @@
 const logger = require("../../../logger");
-const { createSuccessResponse } = require("../../../response");
+const prisma = require("../../../model/data/prismaClient");
+const {
+  createSuccessResponse,
+  createErrorResponse,
+} = require("../../../response");
 
 /**
- * Update an asset by it's id
+ * Update an asset by its id
  */
-
-// eslint-disable-next-line no-unused-vars
 module.exports = async (req, res, next) => {
-  logger.debug(
-    { user: req.user, id: req.params.id },
-    `received request: PATCH /v1/assets/:assetId`,
-  );
+  try {
+    const assetId = Number(req.params.assetId);
```

Comment:
issue: The asset ID is not a number, it's a UUID. We can use Zod for validation:

https://zod.dev/?id=strings

```js
z.string().uuid();
```